integration/v7/isolated/share_service_command_test.go

```diff
@@ -0,0 +1,77 @@
+package isolated
+
+import (
+	"code.cloudfoundry.org/cli/integration/helpers"
+	//"code.cloudfoundry.org/cli/integration/helpers/servicebrokerstub"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("share-service command", func() {
+	const shareServiceCommand = "v3-share-service"
+
+	FDescribe("help", func() {
```

Comment:
Focused test. `ginkgo blur` will fix it.

Reply:
I had made the change but forgot to commit it 👍